README.md

```diff
@@ -91,6 +91,17 @@ Binaries are available from [Netflix OSS Maven](https://artifacts.netflix.net/ne
 
 [Access here other Conductor related projects made by the community!](/RELATED.md) - Backup tool, Cron like workflow starter, Docker containers...
 
+## Client SDKs
```

Comment:
Community contributions are listed in RELATED.md. Please move these links to that file.